sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Generated/Models/TypesBatchResponse.Serialization.cs

```diff
@@ -13,60 +13,5 @@ namespace Azure.IoT.TimeSeriesInsights
 {
     public partial class TypesBatchResponse
     {
-        internal static TypesBatchResponse DeserializeTypesBatchResponse(JsonElement element)
-        {
-            Optional<IReadOnlyList<TimeSeriesTypeOperationResult>> @get = default;
-            Optional<IReadOnlyList<TimeSeriesTypeOperationResult>> put = default;
-            Optional<IReadOnlyList<TimeSeriesOperationError>> delete = default;
-            foreach (var property in element.EnumerateObject())
-            {
-                if (property.NameEquals("get"))
-                {
-                    if (property.Value.ValueKind == JsonValueKind.Null)
-                    {
-                        property.ThrowNonNullablePropertyIsNull();
-                        continue;
-                    }
-                    List<TimeSeriesTypeOperationResult> array = new List<TimeSeriesTypeOperationResult>();
-                    foreach (var item in property.Value.EnumerateArray())
-                    {
-                        array.Add(TimeSeriesTypeOperationResult.DeserializeTimeSeriesTypeOperationResult(item));
-                    }
-                    @get = array;
-                    continue;
-                }
-                if (property.NameEquals("put"))
-                {
-                    if (property.Value.ValueKind == JsonValueKind.Null)
-                    {
-                        property.ThrowNonNullablePropertyIsNull();
-                        continue;
-                    }
-                    List<TimeSeriesTypeOperationResult> array = new List<TimeSeriesTypeOperationResult>();
-                    foreach (var item in property.Value.EnumerateArray())
-                    {
-                        array.Add(TimeSeriesTypeOperationResult.DeserializeTimeSeriesTypeOperationResult(item));
-                    }
-                    put = array;
-                    continue;
-                }
-                if (property.NameEquals("delete"))
-                {
-                    if (property.Value.ValueKind == JsonValueKind.Null)
-                    {
-                        property.ThrowNonNullablePropertyIsNull();
-                        continue;
-                    }
-                    List<TimeSeriesOperationError> array = new List<TimeSeriesOperationError>();
-                    foreach (var item in property.Value.EnumerateArray())
-                    {
-                        array.Add(TimeSeriesOperationError.DeserializeTimeSeriesOperationError(item));
```

Comment:
Earlier it threw an exception here while deleting types

"{"The requested operation requires an element of type 'Object', but the target element has type 'Null'."}"